components/toast/index.css

```diff
@@ -10,18 +10,60 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+/* toast/index.css
+ * 
+ * .spectrum-Toast::after is the Focus ring
+ *
+ * Component naming template:
+ * `--system-component-style-description-modifier`
+ *
+ * Example:
+ * `--spectrum-toast-spacing-text-to-button-horizontal`
+ */
+
 .spectrum-Toast {
-  --spectrum-toast-icon-padding-y: var(--spectrum-global-dimension-size-85);
-  --spectrum-toast-neutral-content-padding-top: var(
-    --spectrum-global-dimension-size-65
-  );
-  --spectrum-toast-content-padding-bottom: var(
-    --spectrum-global-dimension-size-65
-  );
-
-  --spectrum-toast-button-margin-right: var(
-    --spectrum-global-dimension-size-130
-  );
+  /* Hardcoded variables */
+  --spectrum-toast-font-weight: 400;
+
+  /* Size */
+
+  --spectrum-toast-font-size: var(--spectrum-font-size-100);
+  --spectrum-toast-corner-radius: var(--spectrum-corner-radius-100);
+
+  --spectrum-toast-line-height: var(--spectrum-line-height-100);
+  --spectrum-toast-line-height-CJK: var(--spectrum-CJK-line-height-100);
```

Comment:
We may want to use for CJK support the lang-selector:
```CSS
 &:lang(ja),
  &:lang(zh),
  &:lang(ko) {
    line-height: var(--mod-toast-line-height-CJK, var(----spectrum-toast-line-height-CJK));
  }

Reply:
I used the lang selector below in the `.spectrum-Toast-content` styles. Is that what you meant?

Reply:
maybe we should used that when referencing CJK line-heights? @pfulton what do you suggest?

Reply:
@bernhard-adobe could you clarify? I think @yosevu implemented it as I'd expect [here](https://github.com/adobe/spectrum-css/blob/3e3807d310b77681cd07660e10bbd970aedfcfdd/components/toast/index.css#L138-L142).

@yosevu You may have a typo on line 138: `&:lang(js)` should be `&:lang(ja)`, right?

Also, can we lowercase `CJK` in the custom property name, please?

Reply:
@pfulton could we lower-case CJK in core tokens name, please ;)?

Reply:
Ah thanks. Updated in e3f4a568096fb55054c773f48d783c9a00994d2b